web/templates/package/index.html.eex

```diff
@@ -8,7 +8,7 @@
     <ul>
       <%= for letter <- @letters do %>
       <li>
-        <a href="<%= package_path(HexWeb.Endpoint, :index, letter: letter) %>">
+        <a href="<%= package_path(HexWeb.Endpoint, :index, letter: letter, sort: @sort) %>">
```

Comment:
Does it make sense to support sorting on anything different than name when using letters? Maybe the selector should be removed?